### PR TITLE
feat: Added the ability to specify a custom resource group when deploying a private endpoint

### PR DESCRIPTION
**Pull request by AlexanderSehr**
# Overview/Summary

- Added the ability to specify a custom resource group when deploying a private endpoint
- An example deployment can be found [here](https://github.com/Azure/bicep-registry-modules/issues/1190#issuecomment-2013104583)

> Note: While possible to implement, I opted to ask the user for the Resource Group name, as opposed to the Resource Group resource Id as it drastically increases the complexity of the implementation and may also make providing a custom resource group confusing. I would suggest to only change this if ever requested.

### CODE REVIEW

File: docs/static/includes/interfaces/int.pe.input.bicep [hunk 1]
```diff
@@ -1,17 +1,18 @@
 privateEndpoints: {
   {
-    roleAssignments : [...]
     lock: 'CanNotDelete'
-    tags: {...}
+    tags: {
+      'hidden-title': 'This is visible in the resource name'
+    }
     service: 'vault'
```
Comment (eriqua) [inline docs/static/includes/interfaces/int.pe.input.bicep:7]: Is this an example or a schema? if the latter, should we generalize the service value?

Comment (AlexanderSehr) [reply docs/static/includes/interfaces/int.pe.input.bicep:7]: It's the former, an example: 
image

Comment (eriqua) [reply docs/static/includes/interfaces/int.pe.input.bicep:7]: Thanks for elaborating, makes sense. Resolving

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**